tests/test_bulk_update.py

```diff
@@ -45,6 +45,8 @@ def test_values(self):
         self.assertEqual(1, bulk_update(TestModel, {(2, 'test2'): {'int_field': 2}}, key_fields=('id', 'name')))
         self.assertEqual(1, bulk_update(TestModel, {('test3',): {'int_field': 2}}, key_fields='name'))
 
+        self.assertEqual(1, bulk_update(TestModelWithSchema, [{'id': 1, 'name': 'abc'}]))
```

Comment:
Duplicate: It is not `InputFormat` test. It doesn't check validation issues.
What does it check? I'd rather add it to `TestSimple` class as separate test method.